src/ucp/core/ucp_context.h

```diff
@@ -77,6 +77,8 @@ typedef struct ucp_context_config {
     unsigned                               max_rndv_lanes;
     /** Estimated number of endpoints */
     size_t                                 estimated_num_eps;
+    /** Estimated number of endpoints per node */
```

Comment:
we need elaborate somewhere that in this case endpoint per node == process per node. Or maybe just leave it as a process

Reply:
re-worded to process per node